vocabularies/qesd-obs-props.ttl

```diff
@@ -0,0 +1,8937 @@
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix qudt: <http://qudt.org/schema/qudt/> .
+@prefix quantitykind: <http://qudt.org/vocab/quantitykind/> .
+
+<https://linked.data.gov.au/def/qld-obsprop> a skos:ConceptScheme ;
+    dcterms:created "2021-06-06"^^xsd:date ;
+    dcterms:modified "2021-06-06"^^xsd:date ;
+    dcterms:creator <https://linked.data.gov.au/org/des> ;
+    dcterms:publisher <https://linked.data.gov.au/org/gsq> ;
+    skos:prefLabel "QESD Observed Properties vocabulary"@en ;
+    dcterms:provenance "Includes terms from TERN Core Vocabulary (https://linkeddata.tern.org.au/viewer/tern)"@en ; 
+    skos:definition "A set of environmental parameters that can be obervered, and are of relevance to Queensland Environmental Science Data (QESD)."@en ;
+    skos:hasTopConcept 
+        <http://linked.data.gov.au/def/tern-cv/123f2efe-3891-4764-9047-609c8b30e925>,
+        <http://linked.data.gov.au/def/tern-cv/29bd4356-5638-49c0-8d3e-57598ac2ea6c>,
+        <http://linked.data.gov.au/def/tern-cv/2ffe4363-dea1-4188-9f8f-835983d47fe2>,
+        <http://linked.data.gov.au/def/tern-cv/41a27c10-6dff-4ddc-9f8b-14e87fafc786>,
+        <http://linked.data.gov.au/def/tern-cv/45ae73d8-92bd-4261-a05f-7bfac228e1f7>,
+        <http://linked.data.gov.au/def/tern-cv/50b9def1-660b-4274-b10e-6dbe57243305>,
+        <http://linked.data.gov.au/def/tern-cv/68add852-0a47-4412-8f8c-52bd8d0151b1>,
+        <http://linked.data.gov.au/def/tern-cv/6e2ede4b-1e6f-4ea0-93d9-6597c5666d62>,
+        <http://linked.data.gov.au/def/tern-cv/6f726a59-c330-44eb-898e-7d42d5fbdada>,
+        <http://linked.data.gov.au/def/tern-cv/777773d0-cfb4-44ff-85e2-08b48e10ac52>,
+        #<http://linked.data.gov.au/def/tern-cv/8e7a785a-8c57-4415-855f-cb6d35ad58bb>,
+        <http://linked.data.gov.au/def/tern-cv/8fa1c4bd-73b3-432a-ab49-9ee89928e6d9>,
+        <http://linked.data.gov.au/def/tern-cv/99bd0128-54d4-4a2b-8a2c-48010eb07d26>,
+        <http://linked.data.gov.au/def/tern-cv/9e03a73d-8e89-4187-9f96-0306aad93458>,
+        <http://linked.data.gov.au/def/tern-cv/bb5c1354-8d70-47b2-b333-87e3bf1157d6>,
+        <http://linked.data.gov.au/def/tern-cv/c_012ebf32>,
+        <http://linked.data.gov.au/def/tern-cv/d62a4664-89a1-49a1-9ef0-f30ee2c6d22b> ;
+    skos:narrower <http://linked.data.gov.au/def/tern-cv/0986ce8c-4a8c-4132-91f9-b7380d45ce73>,
+        <http://linked.data.gov.au/def/tern-cv/87ed6ce4-3354-4a3f-ba15-2b7cb0caf3da> ;
```

Comment:
Q: How is skos:narrower different to skos:hasTopConcept for the concept scheme?
Should the former not appear in the list of top concepts for the vocabulary?

Reply:
I have updated. Was following the pattern from TERN - but since we have our own ConceptScheme we can make this tweak.